renovate-changesets/index.ts

```diff
@@ -124,11 +126,18 @@ async function main() {
     ),
   );
 
+  // Filter out bumps where `changes` is empty
+  const filteredBumps = bumps.filter(({ changes }) => changes.size > 0);
+
+  if (filteredBumps.length < 0) {
```

Comment:
this is always false

Reply:
Oh yeah, I've changed it to filteredBumps.length === 0